js/src/decorator/tinyMCE.js

```diff
@@ -46,4 +46,30 @@ function tinyMCEDecorator( editor ) {
 	return markTinyMCE.bind( null, editor );
 }
 
-module.exports = tinyMCEDecorator;
+/**
+ * Returns whether or not the editor has marks
+ *
+ * @param {tinyMCE.Editor} editor The editor.
+ * @returns {boolean} Whether or not there are marks inside the editor.
+ */
+function editorHasMarks( editor ) {
+	var content = editor.getContent({ format: 'raw' });
+
+	return -1 !== content.indexOf( '<yoastmark' );
+}
+
+/**
+ * Removes marks currently in the given editor
+ *
+ * @param {tinyMCE.Editor} editor The editor to remove all marks for.
+ */
+function editorRemoveMarks( editor ) {
+	tinyMCEDecorator( editor )( null, [] );
```

Comment:
Perhaps a line of parameter description couldn't hurt here. Looks like magic :rabbit: